mmaction/datasets/pipelines/augmentations.py

```diff
@@ -42,6 +42,209 @@ def _init_lazy_if_proper(results, lazy):
         assert 'lazy' not in results, 'Use Fuse after lazy operations'
 
 
+@PIPELINES.register_module()
+class Imgaug:
+    """Imgaug augmentation.
```

Comment:
What's the input & output type of this op? uint8 np array? float32? float tensor?

Reply:
Different augmenters support different dtyps. Overall, `uint8` supports all augmenters.

Details could be found [here](https://imgaug.readthedocs.io/en/latest/source/dtype_support.html)

Reply:
I care about the type change. If we input an uint8, will the output be guaranted to be uint8? Or the output will be float64 regardless of input?

Reply:
To be more specific, we do not want the output dtype to change due to random switches in the processing pipeline.

Reply:
Based on my experience, dype won't change. But I cannot find any docs about this.

Reply:
https://github.com/aleju/imgaug/issues/110#issuecomment-375043766
The above comment(2018.5) shows that most of the augmenters will preserve input dtype. And we should use uint8 images as input.

Reply:
Okay, this info can be added to docstring